numpy/lib/function_base.py

```diff
@@ -108,20 +115,35 @@ def rice(x):
         The number of bins is proportional to the cube root of data size (asymptotically optimal)
         Depends only on size of the data
         """
-        return np.ceil(2 * x.size ** (1.0 / 3))
+        return np.ceil(2 * np.cbrt(x.size))
```

Comment:
Running `ufunc`s on scalars is much slower than just doing the math, so I would suggest not making this change (and the similar one below). Some timings:
```
In [2]: %timeit np.cbrt(100)
1000000 loops, best of 3: 1.29 µs per loop

In [3]: %timeit 100 ** (1./3.)
100000000 loops, best of 3: 11.5 ns per loop
```